### ISSUE TITLE
Bad URL used for box 364 in vagrant cloud

### ISSUE THREAD

**Issue by sykesm** [cloudfoundry/bosh-lite#196]
Attempting to use the latest bosh-lite fails with

``` shell
$ vagrant up
Bringing machine 'default' up with 'vmware_fusion' provider...
==> default: Box 'cloudfoundry/bosh-lite' could not be found. Attempting to find and install...
    default: Box Provider: vmware_desktop, vmware_fusion, vmware_workstation
    default: Box Version: 364
==> default: Loading metadata for box 'cloudfoundry/bosh-lite'
    default: URL: https://vagrantcloud.com/cloudfoundry/bosh-lite
==> default: Adding box 'cloudfoundry/bosh-lite' (v364) for provider: vmware_desktop
    default: Downloading: https://vagrantcloud.com/cloudfoundry/boxes/bosh-lite/versions/4/providers/vmware_desktop.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

The requested URL returned error: 403 Forbidden
```

It appears that an incorrect url (`http://d3a4sadvqj176z.cloudfront.net//bosh-lite-vmware-ubuntu-trusty-364.box`) with two slashes was associated with the box definition.  This causes Vagrant to get a bit upset as AWS rejects the `GET` request with a 403.

**Comment by dzlab** [comment]
I'm having the same error:
```
dzlab@ubuntu-server:~/workspace$ uname -a
Linux ubuntu-server 3.13.0-32-generic #57-Ubuntu SMP Tue Jul 15 03:51:08 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

dzlab@ubuntu-server:~/workspace$ git clone https://github.com/cloudfoundry/bosh-lite.git
dzlab@ubuntu-server:~/workspace$ cd bosh-lite
dzlab@ubuntu-server:~/workspace/bosh-lite$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* The following settings shouldn't exist: box_version
* The box 'cloudfoundry/bosh-lite' could not be found.
```
I tried to comment the `override.vm.box_version` but then just the first error disappear, it still can't find the 'cloudfoundry/bosh-lite' box.

**Comment by cppforlife** [comment]
@dzlab can you open a new issue for better tracking? also include vagrant version.

**Comment by dzlab** [comment]
I had an old vagrant 1.4.3, i fixed this but upgrading to 1.6.3
